src/generators/vcf_file_generator.py

```diff
@@ -36,14 +36,14 @@ class VcfFileGenerator:
 ##INFO=<ID=transcriptLevelConsequence,Number=.,Type=String,Description="VEP consequence of the variant on the Transcript">
 ##INFO=<ID=geneImpact,Number=1,Type=String,Description="Variant impact scale for Gene">
 ##INFO=<ID=transcriptImpact,Number=.,Type=String,Description="Variant impact scale for Transcript">
-##INFO=<ID=alleleSymbol,Number=1,Type=String,Description="The human readable name of the Allele">
+##INFO=<ID=allele_symbols,Number=.,Type=String,Description="The human readable name of the Allele">
```

Comment:
this needs to match what is below or things will explode . . so probably `alleleSymbols`

Reply:
I am looking at these two fields needing to match what is on line 1174 and 175. I think they do. I have run it through a VCF validator that from my experience will point out if they don't line up. Let m know if you find otherwise.